docs/querying/sql.md

```diff
@@ -391,6 +391,26 @@ like `100` (denoting an integer), `100.0` (denoting a floating point value), or
 timestamps can be written like `TIMESTAMP '2000-01-01 00:00:00'`. Literal intervals, used for time arithmetic, can be
 written like `INTERVAL '1' HOUR`, `INTERVAL '1 02:03' DAY TO MINUTE`, `INTERVAL '1-2' YEAR TO MONTH`, and so on.
 
+## SET statements
+
+Druid SQL over HTTP supports including 0 or more `SET` statements separated by `;` preceding the statement to execute
+in order to assign [SQL query context parameter values](../querying/sql-query-context.md).
```

Comment:
This should clarify that `SET` statements only affect the immediately-following non-`SET` statement, and must be issued in the same HTTP call as the non-`SET` statement. Otherwise people may assume that `SET` affects all future statements they issue (kind of nonsensical since there are no sessions in our HTTP API, but people may assume it anyway).